README.md

```diff
@@ -1,166 +1,171 @@
-
-```html
-<script>
+</script>  
+```  
+  
+## isAuthenticated  
+Use isAuthenticated to check if a user is authenticated. It returns true if the user is authenticated, otherwise false.
+
+```html  
+<script>  
 export default {
-  name: 'login-shortcut-check',
-
+  name: 'auhtenticated-check',
   methods: {
     login () {
-      if (this.$isAuthenticated() !== true) {
-        this.$login()
+      if (this.$gapi.isAuthenticated()) {  
+        this.$login() 
+          .then(() => {
+            // Any actions needed after login
+          }
+          .catch(err => {
+ 	        // Handle errors
+          }
       }
     }
   }
-
 }
-
-</script>
+</script>  
 ```
-
-## refreshToken
-
-This will shortcut getting a refresh token from Google, this should be placed in your App.vue on the created page and run on a timer of 45min
-
-```html
-<script>
-  name: 'App'
-
+## refreshToken  
+This will shortcut getting a refresh token from Google, this should be placed in your App.vue on the created page and run on a timer of 45min  
+  
+```html  
+<script>  
+export default {
+  name: 'App',
   created () {
-  try {
-    // NOTE: 45min refresh policy is what google recommends
-    window.setInterval(this.$refreshToken(), 1000 * 60 * 45)
-  } catch (e) {
-    console.error(e)
+    try {
+      // NOTE: 45 min refresh policy is what Google recommends
+      window.setInterval(this.$refreshToken(), 1000 * 60 * 45)  
+    } catch (e) {
+      console.error(e)
+    }
   }
-
 }
-</script>
-```
-
-## grantOfflineAccess
+</script>  
+```  
+## grantOfflineAccess  
+To use Google services on behalf of a user when the user is offline, a hybrid server-side flow must be used where a user authorizes the application on the client side to retrieve a special one-time authorization code that is sent and stored on a back-end server. The back-end server exchanges this one-time-use code to acquire its own access and refresh tokens from Google for the server to be able to make its own API calls, which can be done while the user is offline. This one-time code flow has security advantages over both a pure server-side flow and over sending access tokens to the back-end server.  
+  
+One-time codes have several security advantages. Google provides tokens directly to your server without any intermediaries. This one-time code can only be used together with the Client Secret. **Keep your Client Secret secret!**
 
 This will retrieve an offline access token. This offline access token can be used server side to access the users data.
-
-```html
-<script>
-  name: 'App'
-
+  
+```html  
+<script>  
+  name: 'offline-access-code',
   methods: {
-      getOfflineAccessCode () {
-        this.$grantOfflineAccess()
-          .then(() => {
-            const offlineAccessCode = this.$getOfflineAccessCode();
-            // Send the offline access token to the backend server
-          }
-      }
-
+    getOfflineAccessCode () {
+      this.$grantOfflineAccess()
```

Comment:
should be consistent in the documentation, this should be called from this.$gapi.getOfflineAccessCode

Reply:
Yes you are right. Missed this one. Thanks for catching it.